src/main/java/nl/tudelft/pixelperfect/gamestates/PlayState.java

```diff
@@ -0,0 +1,86 @@
+package nl.tudelft.pixelperfect.gamestates;
+
+import com.jme3.scene.Spatial;
+import jmevr.app.VRApplication;
+import nl.tudelft.pixelperfect.Game;
+import nl.tudelft.pixelperfect.Spaceship;
+import nl.tudelft.pixelperfect.event.EventScheduler;
+import nl.tudelft.pixelperfect.gui.GameHeadsUpDisplay;
+
+/**
+ * Created by woute on 5/23/2016.
```

Comment:
Can you please refactor alle these class scoped javadoc comments to be consistent with the rest of the code? It should state in a few sentences what the purpose and responsibilities of the class are. Then after a white line there should be an author tag followed by your full name.